transport/server.cc

```diff
@@ -683,7 +683,7 @@ client_data cql_server::connection::make_client_data() const {
     cd.protocol_version = _version;
     cd.driver_name = _client_state.get_driver_name();
     cd.driver_version = _client_state.get_driver_version();
-    if (const auto user_ptr = _client_state.user(); user_ptr) {
+    if (const auto &user_ptr = _client_state.user(); user_ptr) {
```

Comment:
Our coding style sticks the & to auto, not the variable.

Reply:
fixed